src/NewsService.Data/NewsRepository.cs

```diff
@@ -16,19 +16,37 @@ public class NewsRepository : INewsRepository
     private readonly IDataProvider _provider;
     private readonly IHttpContextAccessor _httpContextAccessor;
 
-    public NewsRepository(
-      IDataProvider provider,
-      IHttpContextAccessor httpContextAccessor)
+    private IQueryable<DbNews> CreateFindPredicates(
```

Comment:
1-method doesnt create anything. 2-strange naming. what for predicates in naming? predicate - it method that return bool. thats all. in c# you have realized delegate - Predicate<T>.

Reply:
Removed this method and moved everything to Find

Reply:
done